### ISSUE TITLE
Glyph of Sunder Armor

### ISSUE THREAD

**Issue by warcraftgeekygamer** [dalaranwow/dalaran-wow#4825]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Glyph of Sunder Armor is not working correctly.

**Tell us what happens in detail (Where, When, What, Why, Who):** Currently as a Protection Warrior with Glyph of Sunder Armor, when you use Devastate on a mob the Sunder Armor effect is not applied to a nearby mob.

**Expected behaviour**: When a Protection Warrior has Glyph of Sunder Armor, and use Devastate on a mob, the Sunder Armor effect is supposed to be applied to a nearby mob.

**Steps to reproduce the problem**:

1. Create a Protection Warrior
2. Obtain level 50 or higher
3. Spec into Devastate
4. Use Glyph of Sunder Armor
5. Look for a group of mobs, two or three should suffice
6. Use Devastate on one mob
7. Notice that the Sunder Armor effect is not applied to one of the nearby mobs

**Include proofs for this behaviour**

wowhead link: http://www.wowhead.com/forums&topic=110459/glyph-of-sunder-armor
wowwiki link: http://wowwiki.wikia.com/wiki/Glyph_of_Sunder_Armor

According to wowhead it appears the spellid is now a "charred glyph" (43427). However, there are comments that support my bug report. Please reference the two additional links below.

http://www.wowhead.com/item=43427/charred-glyph#comments
http://www.wowhead.com/item=43427/charred-glyph

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by AngryJim** [comment]
I can confirm I have also seen this issue, but would like to add that occasionally the very first use of Devastate will apply a single sunder to a secondary target, but not every single time, and no further applications of sunder get applied upon further use of devastate.

**Comment by TeridanDW** [comment]
Hello @warcraftgeekygamer, thank you for the report.
I'm going to close this report, because I think @Rushor fixed this issue in #4235.
If there is still a problem after #4235 is on live server, will reopen this report.
